mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.feature/feature.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.eclipse.mylyn.gitlab.feature"
+      label="Mylyn Tasks Connector: Gitlab Preview for gitlab.com only"
+      version="4.0.0.qualifier"
+      provider-name="Eclipse Mylyn"
+      plugin="org.eclipse.mylyn.gitlab.core">
```

Comment:
please reference standard license feature and remove entries with "www.example.com"

Reply:
Does with next commit